system-x/services/hashicorp-vault/src/main/java/software/tnb/hashicorp/vault/account/HashicorpVaultAccount.java

```diff
@@ -0,0 +1,17 @@
+package software.tnb.hashicorp.vault.account;
+
+import software.tnb.common.account.Account;
+
+public class HashicorpVaultAccount implements Account {
+    private static final String DEFAULT_TOKEN = "myTestToken";
+
+    private String token = DEFAULT_TOKEN;
+
+    public String getToken() {
```

Comment:
use the record style getter to match other accounts, e.g. rename this to `token()`

Reply:
can I refactor it as a java `record` directly?

Reply:
since you have a default value and a setter, then I'd say no

Reply:
I was not using the setter, and with the latest version you can always do something like

`new HashicorpVaultAccount("myToken")`